utils/build/docker/nodejs/nextjs/.eslintrc.json

```diff
@@ -21,7 +21,7 @@
     "no-console": 0,
     "prefer-const": 2,
     "object-curly-spacing": [2, "always"],
-    "import/no-extraneous-dependencies": 2,
+    "import/no-extraneous-dependencies": "off",
```

Comment:
why is that needed ?

Reply:
Because of this line : https://github.com/DataDog/system-tests/pull/2801/files#diff-963f90808907fc274dfda97af25b1adf204006b7c4aba7aa22c0119e52fa10e9R3

I don't know how to disable the rule only for this line, so I went lazy.

Reply:
I added a suggestion to that line

Reply:
and revert this one
```suggestion
    "import/no-extraneous-dependencies": 2,
```